blocks/tags/tags.css

```diff
@@ -0,0 +1,65 @@
+.section.tags-container {
+  margin-bottom: 5rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.section.tags-container>.section-container {
+  border-top: 1px solid #3c465033;
+  padding-top: 1.75rem;
+  gap: 2rem;
+}
+
+.section.tags-container>.section-container:has(p) {
```

Comment:
This looks a little awkward, would it make sense to add a class that you can just reference instead of using the `has()` function?